cumulusci/tasks/bulkdata/step.py

```diff
@@ -287,16 +287,16 @@ def _csv_generator(self, records):
         content = io.StringIO()
         writer = csv.writer(content)
         writer.writerow(self.fields)
-
         content.seek(0)
         yield content.read().encode("utf-8")
+        content.truncate(0)
```

Comment:
@davidmreed TIL about truncate :)

Reply:
Stack Overflow scared me into believing it was not performant. Lesson learned - Stack Overflow lies; do the benchmarking yourself.